src/Multigraphs/multigraph_adjlist.jl

```diff
@@ -43,40 +46,40 @@ function Multigraph(adjmx::AbstractMatrix{U}) where {U<:Integer}
             end
         end
     end
-    Multigraph(adjlist)
+    Multigraph{Int}(adjlist, m)
 end
 Multigraph(n::T) where {T<:Integer} = Multigraph(Dict(zip(T(1):n, [T[] for _ = 1:n])))
 Multigraph(g::SimpleGraph{T}) where {T<:Integer} = Multigraph(Dict(zip(T(1):nv(g), LightGraphs.SimpleGraphs.fadj(g))))
 
-copy(mg::Multigraph) = Multigraph(deepcopy(mg.adjlist))
+copy(mg::Multigraph{T}) where {T} = Multigraph{T}(deepcopy(mg.adjlist), mg._idmax)
```

Comment:
```suggestion
copy(mg::Multigraph{T}) where {T} = Multigraph{T}(copy(mg.adjlist), mg._idmax)
```

I don't think `deepcopy` is necessary, is it?